modules/rostests/apitests/user32/SW_FORCEMINIMIZE.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     Test for ShowWindow.WS_FORCEMINIMIZE
+ * COPYRIGHT:   Copyright 2025 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#include "precomp.h"
+#include <versionhelpers.h>
+
+#define RED RGB(255, 0, 0)
+
+static HANDLE g_hThread = NULL;
+
+static COLORREF CheckColor(VOID)
+{
+    HDC hDC = GetDC(NULL);
+    COLORREF color = GetPixel(hDC, 100, 100);
+    ReleaseDC(NULL, hDC);
+    return color;
+}
+
+static DWORD WINAPI
+ThreadFunc(LPVOID arg)
+{
+    BOOL ret;
+    HWND hwnd = (HWND)arg;
+    DWORD style, exstyle;
+
+    Sleep(100);
```

Comment:
Is this to ensure that the window is created before running the rest of the code?
If you start the thread instead in `Test_WS_FORCEMINIMIZE()` after the `CreateWindowEx` call, window creation would be already ensured by the time the thread is started, without the need for this Sleep...

Reply:
It's for `CheckColor`. Without this `Sleep`, `CheckColor` can fail.